lighthouse-core/test/fraggle-rock/api-test-pptr.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env jest */
+
+const path = require('path');
+const lighthouse = require('../../fraggle-rock/api.js');
+const puppeteer = require('puppeteer');
+const StaticServer = require('../../../lighthouse-cli/test/fixtures/static-server.js').Server;
+
+jest.setTimeout(90e3);
+
+describe('Fraggle Rock API', () => {
+  /** @type {InstanceType<StaticServer>} */
+  let server;
+  /** @type {import('puppeteer').Browser} */
+  let browser;
+  /** @type {import('puppeteer').Page} */
+  let page;
+
+  beforeAll(async () => {
+    server = new StaticServer();
+    await server.listen(0, '127.0.0.1');
+    browser = await puppeteer.launch({
+      headless: true,
+    });
+  });
+
+  beforeEach(async () => {
+    page = await browser.newPage();
+  });
+
+  afterEach(async () => {
+    await page.close();
+  });
+
+  afterAll(async () => {
+    await browser.close();
+    await server.close();
+  });
+
+  describe('snapshot', () => {
+    beforeEach(() => {
+      server.setBaseDir(path.join(__dirname, '../fixtures/fraggle-rock/snapshot-basic'));
```

Comment:
nit: use `${...}/${...}`

Reply:
pending results of https://github.com/GoogleChrome/lighthouse/pull/11748#discussion_r534434654